Content.Server/GameObjects/Components/Sound/EmitSoundOnLandComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Shared.Interfaces.GameObjects.Components;
+using Robust.Shared.Audio;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Player;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Sound
+{
+    /// <summary>
+    /// Simple sound emitter that emits sound on land
+    /// </summary>
+    [RegisterComponent]
+    public class EmitSoundOnLandComponent : Component, ILand
+    {
+        /// <inheritdoc />
+        ///
+        public override string Name => "EmitSoundOnLand";
+
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("sound")] public string? _soundName;
```

Comment:
this component should probably also have support for playing from a soundcollection as well as an individual sound, i.e. what your code in ToysComponent does

Reply:
that way it can actually just be used to replace ToysComponent entirely

Reply:
ok, I'll make the change

Reply:
I'd just use the collection and not both IMO.